Virtual-Keyboard/script.js

```diff
@@ -0,0 +1,145 @@
+const Keyboard = {
+  elements: {
+    main: null,
+    keysContainer: null,
+    keys: document.querySelectorAll(".keyboard__key"),
+  },
+  textArea: {
+    inputField: document.querySelector(".use-keyboard-input"),
+    capslock: false,
+  },
+  init() {
+    this.elements.main = document.createElement("div");
+    this.elements.keysContainer = document.createElement("div");
+    this.elements.main.classList.add("keyboard");
+    this.elements.keysContainer.classList.add("keyboard__keys");
+    document.body.appendChild(this.elements.main);
+    this.elements.main.appendChild(this.elements.keysContainer);
+    Keyboard.createKeys();
+  },
+  createKeys() {
+    const keyLayout = [
+      "1",
+      "2",
+      "3",
+      "4",
+      "5",
+      "6",
+      "7",
+      "8",
+      "9",
+      "0",
+      "backspace",
+      "q",
+      "w",
+      "e",
+      "r",
+      "t",
+      "y",
+      "u",
+      "i",
+      "o",
+      "p",
+      "caps",
+      "a",
+      "s",
+      "d",
+      "f",
+      "g",
+      "h",
+      "j",
+      "k",
+      "l",
+      "enter",
+      "done",
+      "z",
+      "x",
+      "c",
+      "v",
+      "b",
+      "n",
+      "m",
+      ",",
+      ".",
+      "?",
+      "space",
+    ];
+    keyLayout.forEach((key) => {
+      const createIconHTML = (icon_name) => {
+        return `<i class="material-icons">${icon_name}</i>`;
+      };
+      let nextRow = ["backspace", "p", "enter", "?"].includes(key);
+      const keyElement = document.createElement("button");
+      keyElement.classList.add("keyboard__key");
+      fragment = keyElement;
```

Comment:
is it possible - ` fragment = keyElement.classList.add("keyboard__key");` ;)

Reply:
no